src/verify-signature.ts

```diff
@@ -0,0 +1,33 @@
+import * as jose from "jose";
+
+import { createDebug } from "./debug";
+import { getJwksUrl } from "./urls";
+
+const debug = createDebug("verify-signature");
+
+export const verifySignature = async (domain: string, signature: string, rawBody: string) => {
```

Comment:
if this is extracted from middleware, remove that code and use this function there

Reply:
Used code in the existing middleware